inc/site-health.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * Pantheon Site Health Modifications
+ *
+ * @package pantheon
+ */
+
+namespace Pantheon\Site_Health;
+
+// If on Pantheon...
+if ( isset( $_ENV['PANTHEON_ENVIRONMENT'] ) ) {
+	add_filter( 'site_status_tests', __NAMESPACE__ . '\\site_health_mods' );
+	add_filter( 'site_status_tests', __NAMESPACE__ . '\\object_cache_tests' );
+}
+
+/**
+ * Modify the Site Health tests.
+ *
+ * @param array $tests The Site Health tests.
+ * @return array
+ */
+function site_health_mods( $tests ) {
+	// Remove checks that aren't relevant to Pantheon environments.
+	unset( $tests['direct']['update_temp_backup_writable'] );
+	unset( $tests['direct']['available_updates_disk_space'] );
+	unset( $tests['async']['background_updates'] );
+	return $tests;
+}
+
+/**
+ * Add object cache tests.
+ *
+ * @param array $tests The Site Health tests.
+ * @return array
+ */
+function object_cache_tests( $tests ) {
+	$tests['direct']['object_cache'] = [
+		'label' => __( 'Object Cache', 'pantheon' ),
+		'test'  => 'test_object_cache',
+	];
+
+	return $tests;
+}
+
+/**
+ * Check for object cache and object cache plugins.
+ *
+ * @return array
+ */
+function test_object_cache() {
+	if ( ! isset( $_ENV['CACHE_HOST'] ) ) {
+		$result = [
+			'label' => __( 'Redis Object Cache', 'pantheon' ),
+			'status' => 'critical',
+			'badge' => [
+				'label' => __( 'Performance', 'pantheon' ),
+				'color' => 'red',
+			],
+			'description' => sprintf(
+				'<p>%s</p>',
+				__( 'Redis object cache is not active for your site.', 'pantheon' )
+			),
+			'test' => 'object_cache',
+		];
+
+		return $result;
+	}
+
+	$wp_redis_active = is_plugin_active( 'wp-redis/wp-redis.php' );
```

Comment:
I don't believe this is the case for OCP, but my understanding (which could be flawed) is that wp-redis does not need to be active if the drop-in is in place. This is at least alluded to in our docs. If we are going to alert if wp-redis is installed but not active, we may want to adjust language here and/or in the docs.

Reply:
this does not alert if wp-redis is installed but not active. it alerts if wp-redis _is_ installed _and_ active. we get an alert if Redis is installed and _no_ plugins are active, we get an alert if Redis is _not_ installed, and we get a recommendation if wp-redis is used and active.